astro/src/content/docs/apis/_tenant-application-email-configuration.mdx

```diff
@@ -95,7 +96,15 @@ import InlineField from 'src/components/InlineField.astro';
   </APIField>
 
   <APIField name={ props.base_field_name + ".emailConfiguration.properties" } type="String" optional since="1.8.0" renderif={props.is_tenant}>
-    Additional Email Configuration in a properties file formatted String.
+    Custom SMTP configuration properties that may be necessary in some cases. This can contain any Java mail property. It will override anything FusionAuth sets by default.
+
+   <DefaultSMTPConfigurationProperties />
+
+   Here's an example value which overrides these properties; in this case setting both timeout defaults to 5 seconds. 
+
+```
+mail.smtp.timeout=5000\nmail.smtp.connectiontimeout=5000
```

Comment:
Looks like we have `\n` instead of an actual newline here

Reply:
actually, that's the way to send stuff up to the API call. I tested it. If you don't escape the newlines, you get this error message:

```
{"fieldErrors":{},"generalErrors":[{"code":"[invalidJSON]","message":"Invalid JSON in the request body. The property was [unknown]. The error was [Unexpected mapping exception]. The detailed exception was [Illegal unquoted character ((CTRL-CHAR, code 10)): has to be escaped using backslash to be included in string value\n at [Source: (io.fusionauth.http.io.ReaderBlockingByteBufferInputStream); line: 1, column: 75] (through reference chain: io.fusionauth.domain.api.TenantRequest[\"tenant\"]->io.fusionauth.domain.Tenant[\"emailConfiguration\"]->io.fusionauth.domain.EmailConfiguration[\"properties\"])]."}]}%   
```

Reply:
Ah, nice.